server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetVacationResponseMethod.java

```diff
@@ -72,20 +79,25 @@ public SetVacationResponseMethod(VacationRepository vacationRepository, Notifica
         Preconditions.checkArgument(request instanceof SetVacationRequest);
         SetVacationRequest setVacationRequest = (SetVacationRequest) request;
 
+        TimeMetric timeMetric = timeMetricFactory.generate(METHOD_NAME.getName());
         if (!setVacationRequest.isValid()) {
-            return Stream.of(JmapResponse
+            Stream<JmapResponse> responses = Stream.of(JmapResponse
                 .builder()
                 .clientId(clientId)
                 .error(ErrorResponse.builder()
                     .type(INVALID_ARGUMENTS1)
                     .description(INVALID_ARGUMENT_DESCRIPTION)
                     .build())
                 .build());
+            timeLogger.log(timeMetric);
```

Comment:
Why no try / finally here ?

Could we get consistant ?

What if I also want to know response times on invalid requests ?